src/sage/rings/number_field/number_field.py

```diff
@@ -4352,7 +4390,7 @@ def pari_nf(self, important=True):
             RuntimeError: Unable to factor discriminant with trial division
 
         Next, we illustrate the ``maximize_at_primes`` and ``assume_disc_small``
-        parameters of the ``NumberField`` constructor. The following would take
+        parameters of the :func:`NumberField` constructor. The following would take
         a very long time without the ``maximize_at_primes`` option::
 
```

Comment:
`:class:`?

Reply:
This one is really just a factory function, but that's just an implementation detail; so I'll change it as you suggest.